data/scenarios/Tutorials/move.yaml

```diff
@@ -322,13 +398,25 @@ robots:
   #################
   - name: 1P flower
     system: true
-    program: run "scenarios/Tutorials/move_system.sw"
+    program: |
+      def main = \a. return noop end
+      m <- instant (run "scenarios/Tutorials/move_surveil.sw";
+      main [action="P", entity="flower", room=1]);
+      m
   - name: 2P flower
     system: true
-    program: run "scenarios/Tutorials/move_system.sw"
+    program: |
+      def main = \a. return noop end
+      m <- instant (run "scenarios/Tutorials/move_surveil.sw";
+      main [action="P", entity="flower", room=2]);
+      m
   - name: 3P flower
     system: true
-    program: run "scenarios/Tutorials/move_system.sw"
+    program: |
+      def main = \a. return noop end
+      m <- instant (run "scenarios/Tutorials/move_surveil.sw";
+      main [action="P", entity="flower", room=3]);
+      m
```

Comment:
@byorgey this is how `instant` has to be used to not pay for tangible `run`. The result is the rest of the computation which waits and places the wall.